File: android/app/src/main/java/com/xirigo/ecommerce/core/designsystem/component/XGPriceLayout.kt
package com.xirigo.ecommerce.core.designsystem.component

/** Controls the arrangement of sale price and strikethrough in [XGPriceText]. */
enum class XGPriceLayout {
    /** Sale price + strikethrough side-by-side (Row). Default for standard/grid cards. */
    Inline,

    /** Strikethrough above, sale price below (Column). Used for featured/horizontal-scroll cards. */
    Stacked,
}

File: ios/XiriGoEcommerce/Core/DesignSystem/Component/XGPriceLayout.swift
/// Controls the arrangement of sale price and strikethrough in ``XGPriceText``.
///
/// Token source: `components/atoms/xg-price-text.json`.
enum XGPriceLayout {
    /// Sale price + strikethrough side-by-side (HStack). Default for standard/grid cards.
    case inline
    /// Strikethrough above, sale price below (VStack). Used for featured/horizontal-scroll cards.
    case stacked
}
